components/scripts/gradle/04-validate-remote-build-caching-ci-ci.sh

```diff
@@ -311,8 +312,8 @@ EOF
   wait_for_enter
 }
 
-#Overrides config.sh#print_command_to_repeat_experiment
-print_command_to_repeat_experiment() {
+#Overrides config.sh#generate_command_to_repeat_experiment
+generate_command_to_repeat_experiment() {
```

Comment:
I thought about whether or not we could get rid of this function override (ant the override in the equivalent Maven experiment script), but the issue is that this script does set some of the variables that would result in invalid flags being set for this particular script.

I suppose we could add special handling for this script specifically (test if the experiment is Gradle exp 04 and skip evaluating the invalid flags), but that has its own downsides.

Reply:
TBH, I think the override is fine. This particular function is overridden in `exp4-gradle`, `exp5-gradle`, `exp3-maven`, and `exp4-maven`. It would take a lot of special handling to make a single function handle all cases.